config/v1/types_descheduling.go

```diff
@@ -0,0 +1,47 @@
+package v1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// Descheduler holds cluster-wide config information to run the Kubernetes Descheduler
+// and influence its placement decisions. The canonical name for this config is `cluster`.
+type Descheduler struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// spec holds user settable values for configuration
+	// +kubebuilder:validation:Required
+	// +required
+	Spec DeschedulerSpec `json:"spec"`
+	// status holds observed values from the cluster. They may not be overridden.
+	// +optional
+	Status DeschedulerStatus `json:"status"`
+}
+
+type DeschedulerSpec struct {
+	// Policy contains the name of an upstream Descheduler policy configmap in the openshift-kube-descheduler-operator project
+	Policy ConfigMapNameReference `json:"polocy"`
+	// DeschedulingIntervalSeconds is the number of seconds between descheduler runs
+	// +optional
+	DeschedulingIntervalSeconds *int32 `json:"deschedulingIntervalSeconds,omitempty"`
+	// Flags for descheduler.
+	// +optional
+	Flags []string `json:"flags,omitempty"`
+	// Image of the deschduler being managed. This includes the version of the operand(descheduler).
+	Image string `json:"image,omitempty"`
```

Comment:
Isn't that tied to a particular version of the operator? Why we'd want to expose that? We're not doing that anywhere else.

Reply:
It was useful for debugging, but it's probably not really necessary